### PR TITLE
fix: set max-height for typeahead dropdown.

### PR DESCRIPTION
**Pull request by vikram-raj**

### CODE REVIEW

File: projects/ngx-launcher/src/lib/components/targetenvironment-step/target-environment-review.component.html [hunk 1]
```diff
@@ -16,7 +16,7 @@ <h2 class="card-pf-title">
                 placeholder="New Application" type="text" #projectName="ngModel" [(ngModel)]="data.projectName"
                 lowercase validateProjectName required (keyup.enter)="$event.target.blur();">
             </div>
-            <div *ngIf="projectName.invalid && (projectName.dirty || projectName.touched)" class="col-xs-8 col-xs-offset-4 has-error">
```
Comment (edewit) [inline projects/ngx-launcher/src/lib/components/targetenvironment-step/target-environment-review.component.html]: why did you remove this? It only needs to validate when the user changes the field as the prefilled value is always valid

Comment (vikram-raj) [reply projects/ngx-launcher/src/lib/components/targetenvironment-step/target-environment-review.component.html]: Ah, I don't need this changes. Pushed it by mistake. Reverting this changes.

Comment (vikram-raj) [reply projects/ngx-launcher/src/lib/components/targetenvironment-step/target-environment-review.component.html]: Done. https://github.com/fabric8-launcher/ngx-launcher/pull/450/commits/be59bd83167a3c7b028ec0cd4e1b5c38a6eb8df9
